apidoc/LikeButton.yml

```diff
@@ -115,6 +114,16 @@ properties:
     description: |
         This property must be set before rendering the button.
     type: String
+    deprecated:
+        since: 5.0.0
```

Comment:
Please add the "removed: 5.0.0" line here, as it says it is removed in the description.